### PR TITLE
docs: move generator to travis

### PR DESCRIPTION
**Pull request by drjova**
```bash
$ npm run-script docs
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,6 +49,7 @@ install:
 
 after_success:
   - 'npm run-script coverage'
+  - 'npm run-script docs'
```
Comment (jirikuncar) [inline .travis.yml]: imho `npm test` and `npm run-script docs` should be part of `script` section

File: .travis.yml [hunk 2]
```diff
@@ -38,14 +38,20 @@ node_js:
   - '5.0.0'
   - '5.1.0'
 
+env:
+  global:
+    - GH_REF: 'github.com/inveniosoftware/invenio-search-js.git'
+    - GH_TOKEN:
```
Comment (drjova) [inline .travis.yml]: TODO: Add the inveniosoftware token

File: deploy.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+npm i gulp-jsdoc
+npm run-script docs
+
+# go to the out directory and create a *new* Git repo
+cd docs
+git init
+
+# inside this git repo we'll pretend to be a new user
+git config user.name "Travis CI"
+git config user.email "me@drjova.com"
```
Comment (jirikuncar) [inline deploy.sh]: info@invenio-software.org

File: deploy.sh [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+rm -rf node_modules/gulp-jsdoc
+npm i gulp-jsdoc
+npm run-script docs
+
+# go to the out directory and create a *new* Git repo
+cd docs
+git init
+
+# inside this git repo we'll pretend to be a new user
+git config user.name "Travis CI"
```
Comment (jirikuncar) [inline deploy.sh]: invenio-developers

**Pull request merged**